Extend comments on damping parameter (#157)
compgeolab/eql-gradient-boosted: manuscript/content.tex

\noindent where $\lambda$ is a \emph{dimensionless} damping parameter and
regularization is applied on the scaled coefficients $\mathbf{m}$ instead of
$\mathbf{c}$.
Using a dimensionless damping parameter allows us to narrow the range of values
of $\lambda$ that would generate the most accurate predictions, irrespective
of the dataset and its units.
The damping parameter controls the amount of regularization that is be applied:
a large damping value would generate smooth solutions that fail to reproduce
the high frequency components, while a small value would overfit the data,

> failing to produce accurate predictions.

Reviewer: ```suggestion
failing to produce realistic interpolation results.
```

From experience, we recommend searching for suitable $\lambda$ values between
$10^{-6}$ and $10^{4}$ varying by order-of-magnitude, irrespective of the
dataset.

> The choice of the damping and any other hyper-parameter could be done through

Reviewer: ```suggestion
The choice of the damping and other hyper-parameters like the source depth could be done through
```

> validation methods, such as cross-validation algorithms.

Reviewer: ```suggestion
well-established statistical methods, such as cross-validation.
```
